Microsoft.Toolkit.Win32/Microsoft.Toolkit.Win32.UI.Interop.Sample/WrappedRectangle.cs

```diff
@@ -0,0 +1,68 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Microsoft.Windows.Interop.Sample
```

Comment:
Sample code is nice for folks to aplay around with, but there will also need to be documentation for this in the `/docs` folder